### PR TITLE
global sort: fix set tidb_cloud_storage_uri should be redacted

### LINKED ISSUE #48498: global sort set tidb_cloud_storage_uri should be redacted
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. set tidb_cloud_storage_uri with ak, sk, session-token
2. import into data with global sort
3. check the log

### 2. What did you expect to see? (Required)
log should not have the account info
### 3. What did you see instead (Required)
图片

### 4. What is your TiDB version? (Required)
v7.5.0

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: close #48498 

Problem Summary:
As the issue said
### What is changed and how it works?
1. Redact uri in log.
2. Redact uri in show processList.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   
1. In tidb log:
image
3. show processlist;
image

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
redact tidb_cloud_storage_uri in log
```

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Review by lance6716 — APPROVED** [review]

File: pkg/executor/set.go [hunk 1]
```diff
@@ -161,7 +161,11 @@ func (e *SetExecutor) setSysVariable(ctx context.Context, name string, v *expres
 			}
 			return nil
 		})
-		logutil.BgLogger().Info("set global var", zap.Uint64("conn", sessionVars.ConnectionID), zap.String("name", name), zap.String("val", valStr))
+		showValStr := valStr
```
Comment (D3Hunter) [inline pkg/executor/set.go:164]: maybe impl `SensitiveStmtNode`? there're other place might log the sql

**Pull request merged**